packages/core/src/components/accordion/components/accordion-section.interfaces.ts

```diff
@@ -9,3 +9,7 @@ export const stateMap: Record<AccordionSectionState, string> = {
   danger: "fout:",
   error: "fout:",
 };
+
+export interface AccordionSectionToggleEndEvent {
+  open: boolean;
```

Comment:
hier lijkt het `element` te ontbreken. is deze interface nog wel relevant?

Reply:
Nee, die was voor het event op de section.